go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/zclconf/go-cty v1.10.0
 	golang.org/x/mod v0.5.1
 	golang.org/x/oauth2 v0.0.0-20220411215720-9780585627b5
+	golang.org/x/text v0.3.7
```

Comment:
why do we need it? What license does it have?

Reply:
It's needed to fix error reported by linter, as `strings.Title` may not handle unicode correctly.
License is the same as for Go (https://cs.opensource.google/go/x/text/+/master:LICENSE, https://go.dev/LICENSE )

Reply:
can we add ignore to that rule in those two places and not add more third-party dependencies? =)

I'd park this till more holistic go1.18 update.

Reply:
ok. 
it's really not a 3rd party, just "extension pack" from Golang devs...

Reply:
@nfx added ignore instead of using new library